docs/dg/projectManagement.md

```diff
@@ -21,6 +21,27 @@ This page contains information about project management tasks. The target audien
 
 <!-- ==================================================================================================== -->
 
+## Making a release on Github
+
+Before making a release, please check the following prerequisite.
```

Comment:
Should be "please check the following prerequisites:" (plural and colon).

Reply:
Thanks for pointing it out. I will change it accordingly.